emu/src/emu_sensors.c

```diff
@@ -123,11 +129,13 @@ bool checkButtonQueue(buttonEvt_t* evt)
  */
 void emuSensorHandleKey( int keycode, int bDown )
 {
+#ifdef QUIT_EMU_KEY_ENABLED
```

Comment:
What was this change for?

Reply:
It was so I can retain my sanity when using screenshot tools, screen rulers, and magnifiers. `Alt + F4` or `Fn + Alt + F4` work just as well to exit the emu, but if someone really wants `Esc` to exit, they can still define the variable to turn it on.

Reply:
@dylwhich do you care?

Reply:
![image](https://user-images.githubusercontent.com/666399/205517996-9c609c42-571a-4beb-9c7d-ac7456fb658a.png)

![image](https://user-images.githubusercontent.com/666399/205518027-30fce29d-9b7c-4bac-ad00-e73a8f6c3fba.png)

He can still weigh in, but I disabled it after he said I could do it if I wanted to